src/v2/ion/uiSchemaTransformer.js

```diff
@@ -134,6 +134,19 @@ const createUISchema = (remediationValue = [], factors = [], authenticators = []
     const uiSchema = {
       type: 'text',
     };
+
+    if (ionFormField.name === 'rememberMe') { 
+      // For Remember Me checkbox, we need the label only on the right side of it.
+      uiSchema.placeholder = ionFormField.label;
+      uiSchema.label = false;
+      // Separating prop type for Backbone.Model
+      // from html input type
+      uiSchema.modelType = ionFormField.type;
```

Comment:
Note for @haishengwu-okta : I know we discussed this to be a generic field for uiSchema level, however it seems like weird values were getting passed `ionField.type` can be `set` or `object` etc (discovered when i was writing tests), hence kept it to only for rememberMe checkbox.

Reply:
good catch!
which field has type `set` ?

Reply:
now Im thinking we can have generic function call `convertIonTypeToCourageModelType`? thoughts?

Reply:
Yep, it makes sense to have it for every field. Also, makes sense to always keep HTML type separated from Courage Model type.

Reply:
Here are the two objects that will have `modelType` preset to `set`:
https://github.com/okta/okta-signin-widget/blob/52209eaa88564525dda86ca3bc9eb6b3f031e2c9/test/unit/spec/v2/ion/uiSchemaTransformer_spec.js#L356

https://github.com/okta/okta-signin-widget/blob/52209eaa88564525dda86ca3bc9eb6b3f031e2c9/test/unit/spec/v2/ion/uiSchemaTransformer_spec.js#L576

Reply:
But having weird values like `set` makes it hard to have a generic function like that. what do you think? I have also noticed `object` for type. It will be complex to handle such scenarios.

Reply:
`object` make sense. What you have is good for now.

Reply:
can we not check for `ionFormField.type==='boolean'` instead of `ionFormField.name === 'rememberMe'`?

Reply:
We want to target `rememberMe` and hence not checking against `type`. A few reasons - 
* A `boolean` may not always translate to checkbox (dropdown)
* In the case of rememberMe, we don't want label but we want placeholder to have the value from label. (to have the text display only on the right side of the checkbox).

Reply:
> A boolean may not always translate to checkbox (could be dropdown in UI - depends on the implementation)
I don't see how a dropdown component would fit a boolean datatype. A dropdown will have many items in it which more suite an array datatype.

> In the case of rememberMe, we don't want label but we want placeholder to have the value from label. (to have the text display only on the right side of the checkbox).

I would assume that we want all our checkboxes to look the same. Curious why do we think we don't want this same behavior of getting value from label for any other checkbox.

Reply:
I was suggesting to check on `name` cause it feels to me that we might want deal with `boolean` type differently per use case.
Given your comments, maybe we could just check on type `boolean` for now is a good idea.
sorry @anipendakur-okta for back and force.

Reply:
yeah, I think we can check for boolean now given that this is the only use case for it. We can always update our logic later.

I also feel like since we have asked the API to include `type` specifically for the remember me checkbox, we should make use of it to see how it fits our needs on the front-end. If it does not meet our requirements, then we should suggest changes.